src/modules/hub.bicep

```diff
@@ -79,6 +87,60 @@ module dataFactory 'Microsoft.DataFactory/factories/deploy.bicep' = {
   }
 }
 
+resource storageAccountLookup 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
+  name: storageAccount.name
+}
+
+resource blobService 'Microsoft.Storage/storageAccounts/blobServices@2021-06-01' = {
+  parent: storageAccountLookup
+  name: 'default'
+}
+
+resource containers 'Microsoft.Storage/storageAccounts/blobServices/containers@2021-06-01' = [for containerName in containerNames: {
+  parent: blobService
+  name: toLower(containerName)
+  properties: {
+    publicAccess: 'None'
```

Comment:
Is this about anonymous access? Did you verify that exports can still push to `ms-cm-exports` with this? If so, we're good. (FYI: You can test that by creating an export from the same RG. Doesn't need to be much. Just make sure the file gets written without any errors.)

Reply:
Im not sure I can test this without @MSBrett 's changes can I?

Reply:
Works perfectly! Thanks!